uint8 PACKET_TYPE_POINTS = 0
uint8 PACKET_TYPE_HVTABLE = 1

builtin_interfaces/Time stamp
uint8 type # packet type
uint8[] data # packet contents